src/include/execution/compiler/codegen.h

```diff
@@ -272,6 +272,52 @@ class CodeGen {
    */
   ast::Expr *MemberExpr(ast::Identifier lhs, ast::Identifier rhs);
 
+  /**
+   * @return The generated sql NULL bool.
+   */
+  ast::Expr *NullBool();
```

Comment:
Try renaming these `Null*()` functions to `Null*Val()` to indicate these are SQL values?